USAGE.md

```diff
@@ -267,14 +267,45 @@ Policies Leaf NS - - - - - - - - >           group
 (managed by NS user)
 ```
 
-## How to Grant User based Access for LDAP/OIDC?
+## How to Grant User based Access for LDAP/OIDC/JWT?
 
 Example Terraform code is given in [`./docker/terraform/vault-identity-ldap-oidc-user-access.tf`](./docker/terraform/vault-identity-ldap-oidc-user-access.tf).
 
 The LDAP user `kv-writer` is granted read/write privilges for the KV engine in the `tenant` namespace.
 
 The access is only granted for LDAP logins, because the Entity Alias is connected to the LDAP auth backend only.
 
+## How to login using JWT token authentication from CLI?
+This playground also has a JWT authentication backend available. Configuration is basically the same as for the OIDC authentication backend. The difference is that a token fetched from keycloak can be used directly to log into Vault.
```

Comment:
As an OpenID connect newbie I would love to have 1-2 sentences more here regarding the differences in the authentication flow. Maybe using that source here? https://www.keycloak.org/docs/latest/securing_apps/#flows